src/elements/content-sidebar/activity-feed/comment/Comment.js

```diff
@@ -129,22 +130,52 @@ class Comment extends React.Component<Props, State> {
             getMentionWithQuery,
             mentionSelectorContacts,
         } = this.props;
-        const { isConfirmingDelete, isEditing, isInputOpen } = this.state;
+        const { isConfirmingDelete, isEditingInline, isInputOpen } = this.state;
         const createdAtTimestamp = new Date(created_at).getTime();
         const createdByUser = created_by || PLACEHOLDER_USER;
+        const isTask = type === COMMENT_TYPE_TASK;
+        const isEditSupported = isTask; // comment editing not supported
```

Comment:
We usually try to avoid rename-only reassignments. Can we use `isTask` where it's needed?